test/integration/suites/node-attestation/00-setup

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+echo ${ROOTDIR}
+"${ROOTDIR}/setup/x509pop/setup.sh" conf/server conf/ # Throw the agent cert in conf/, it is unused
```

Comment:
we are not using x509pop, you can remove it, or `maybe` update code to test an x509 attestation? just to veirfy another thing than jointoken?

Reply:
This is actually needed just because that script is used to bootstrap the cert
It would be cleaner to have a different script just for bootstrapping